CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## [Unreleased]
 
-- No changes yet.
+- Add `buf beta studio-agent` command which supports running an HTTP(S) server as the Studio agent.
```

Comment:
```suggestion
- Add `buf beta studio-agent` command to support the upcoming Buf Studio.
```

Given that "the Studio agent" doesn't say much about what its for, I think its better to note like this.